test/canisters/canister_user_timer_override/test/conftest.py

```
@@ -0,0 +1,7 @@
+"""The pytest fixtures
+   https://docs.pytest.org/en/latest/fixture.html
+"""
+
+# pylint: disable=missing-function-docstring, unused-import, wildcard-import, unused-wildcard-import, line-too-long, unused-argument
+import pytest
+from icpp.conftest_base import *  # pytest fixtures provided by icpp-pro
```

Ruff (0.15.12)

[error] `from icpp.conftest_base import *` used; unable to detect undefined names

(F403)